types/office-js-preview/index.d.ts

```diff
@@ -1128,8 +1131,98 @@ declare namespace Office {
              * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
              */
             errorMessage?: string;
+            /**
```

Comment:
This could be out of scope of this Review. But the EventCompletedOptions page is getting convoluted. 

i.e. some of these options are valid for Execute Function, others for Spam Reporting, Others for On Send (Darren is adding a bunch for On Send soon). 

Maybe we could divide up the page so that it's organized by type, and what options apply to each type.

Reply:
Thanks for the feedback. Per our discussion, the notes mentioning supported features for each option will suffice for now. I'll work with you to explore other possible formatting options outside of this PR.